frontend/react/style-guide.md

```diff
@@ -400,11 +399,8 @@ src
     }
 
     // good
-    class MyComponentContainer extends Component {
-      render() {
-        const { foo, bar } = this.props;
-        return <MyComponent foo={foo} bar={bar} />
-      }
+    function MyComponentContainer({ foo, bar }) {
+      return <MyComponent foo={foo} bar={bar} />;
```

Comment:
same here

Reply:
If you don't want to keep anything related to classes, I prefer that bad example would be a functional component too.

Reply:
I agree